src/source/Signaling/Signaling.h

```diff
@@ -243,6 +243,7 @@ typedef struct {
     UINT64 getIceConfigTime;
     UINT64 deleteTime;
     UINT64 connectTime;
+    PSignalingStats pSignalingStats;
```

Comment:
ditto. Each individual owner will drive this

Reply:
Got it